packages/forge/src/governance/Governor.sol

```diff
@@ -42,6 +40,8 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor {
     mapping(uint256 => ProposalCore) private _proposals;
     mapping(address => uint256) private _numSubmissions;
```

Comment:
Interesting, and by pack do you mean putting them into like an array of addresses for example?

Oh interesting with the public vars, will look more into this!

Reply:
By packing I mean this described here: https://medium.com/coinmonks/gas-optimization-in-solidity-part-i-variables-9d5775e43dde
In your case, I'm thinking about these three mappings:
```solidity
mapping(address => uint256) public addressTotalVotes;
mapping(address => bool) public addressTotalVotesVerified;
mapping(address => bool) public addressSubmitterVerified;
```
That uint256 and the two bools could be added to a struct, then you could use one mapping for all three.
You could save even more storage if you decreased the size of the uint - since bools are stored on 8 bits, if you changed the uint256 to uint240 or lower, all three could fit into the same slot.

Reply:
Gotcha, alright yeah this is super helpful, thank you!